tests/test_ensemble_tools.py

```diff
@@ -248,7 +258,7 @@ def test_create_dict_sweep(
             parameter = list(parameter_dict.keys())[i]
             scan_range = parameter_dict[parameter]
             para_value = calculate_parameter_value(
-                n_dirs, scan_range[0], scan_range[1], indices[i]
+                scan_range[2], scan_range[0], scan_range[1], indices[i]
             )
```

Comment:
Unpacking `parameter_dict[parameter]`  with something like `start, end, n_dir = parameter_dict[parameter]` would give more meaningful names to different entries and make intention of subsequent `calculate_parameter_value` call clearer.